### ISSUE TITLE
Results are not in agreement with what is mentioned in the paper

### ISSUE THREAD

**Issue by aminfazy** [nwojke/cosine_metric_learning#77]
Using the given trained model(cosine softmax) for MARS dataset feat_test.mat generated using the export mode. When MARS evaluation script(test_mars.m)  is run with this feat_test.mat following result is obtained:
"average of confusion matrix for XQDA:  mAP = 0.278846, r1 precision = 0.351910 "
and rest of the script throws dimension error.
The result mentioned in the paper states otherwise. Please help to understand why there is so much gap.

**Comment by NicoIasCaro** [comment]
Using the default parameters for MARS dataset (cosine softmax, learning_rate=1e-3 ...) I have a rank 1 precision close to 0.6 whereas it is 0.8 in the paper.
My triplet_loss is around 0.5 as in the paper, however all my training metrics fluctuate a lot, I don't know why.
